db/migrate/20170612073558_add_relations_questions.rb

```diff
@@ -0,0 +1,7 @@
+class AddRelationsQuestions < ActiveRecord::Migration
```

Comment:
migration names should be meaningful, this migration could have a better name e.g. `AddUserToQuestion`